.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,3 +3,11 @@
 ### Changes proposed in this pull request
 
 ### Guidance to review
+
+### Checklist
+
+- [ ] Make sure all information from the Trello card is in here
+- [ ] Attach to Trello card
+- [ ] Rebased master
+- [ ] Cleaned commit history
+- [ ] Tested by running locally
```

Comment:
Just to double-check, but I guess that this means not just the tests, but actual manual testing?

Reply:
Yes, running the changes manually to ensure it works.